src/activities/guesscount-summer/GuesscountSummer.qml

```diff
@@ -0,0 +1,374 @@
+        onStop: { Activity.stop() }
+
+
+        JsonParser {
+            id: parser
+            onError: console.error("Guesscount: Error parsing JSON: " + msg);
+        }
+
+        Loader {
+            id: admin
+            active: false
+            sourceComponent: Column {
+                spacing: 10
+                width: parent.width
+                height: parent.height
+
+                Repeater {
+                    id:levels
+                    model: Activity.numberOfLevel
+                    Admin {
+                        id:level
+                        level: modelData+1
+                        width: parent.width
+                        height: parent.height
+                        data: items.data
+                    }
+                }
+            }
+        }
+
+        Rectangle {
+            id: row1
+            width: parent.width
+            height: parent.height/10
+            anchors {
+                top: parent.top
+                topMargin: (parent.height/80)*3
+            }
+            color: "transparent"
+            Rectangle {
+                id: questionNo
+                width: parent.width*0.328;
+                height: parent.height;
+                radius: 20.0;
+                color: "steelblue"
+                anchors {
+                    left: parent.left
+                    leftMargin: parent.width*0.028
+                }
+                GCText {
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    anchors.verticalCenter: parent.verticalCenter
+                    fontSize: mediumSize
+                    text: qsTr("%1/%2").arg(items.sublevel).arg(items.data.length)
+                }
+            }
+            Rectangle {
+                width: parent.width*0.35;
+                height: parent.height;
+                radius: 20
+                color: "orange"
+                anchors {
+                    right: parent.right
+                    rightMargin: parent.width*0.028
+                }
+                GCText {
+                    id: guess
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    anchors.verticalCenter: parent.verticalCenter
+                    fontSize: smallSize
+                    text: qsTr("Guesscount: %1").arg(items.result)
+                }
+            }
+        }
+
+        Column {
+            spacing: 10
+            anchors.top:row1.bottom
+            anchors.topMargin: 10
+            anchors.left: parent.left
+            anchors.leftMargin: 5
+            width: parent.width
+            height: parent.height
+            OperatorRow {
+                id: operatorRow
+                width: parent.width
+                height: parent.height/10
+                mode: items.mode
+                operators: items.levelArr
+                level: items.currentlevel
+            }
+            OperandRow {
```

Comment:
at last level, when there are 5 numbers, the last one is not fully displayed on phone/tablet in portrait mode)